templates/position/position_list.html

```diff
@@ -210,37 +215,74 @@ <h1>Opinions/Positions/Endorsements</h1>
 {#    <p><a href="{% url 'position:position_new' %}">Add New Position</a></p>#}
 
     <script>
+    function displayLoadingBanner() {
+        document.getElementById('FilterPopupMessage').style.display = 'block';
+    }
+
+    function clearSearch() {
+        displayLoadingBanner();
+        document.getElementById('position_search_id').value = '';
+        document.forms['organization_position_choose_election'].submit();
+    }
+
+    function showAllStates() {
+        displayLoadingBanner();
+        document.getElementById('state_code_id').value = '';
+        document.forms['organization_position_choose_election'].submit();
+    }
     $(function() {
         $('#google_civic_election_id').change(function() {
+            displayLoadingBanner();
             this.form.submit();
         });
-    });
-    $(function() {
+
         $('#show_all_elections_id').change(function() {
+            displayLoadingBanner();
             this.form.submit();
         });
-    });
-    {# wv-103: "Friends only" check box for admin users #}
-    $(function() {
+
+        {# wv-103: "Friends only" check box for admin users #}
         $('#show_friends_only_id').change(function() {
+            displayLoadingBanner();
             this.form.submit();
         });
-    });
-    {# end wv-103 #}
-    $(function() {
-        $('#show_statistics_id').change(function() {
+
+        $('#show_this_year_of_elections_id').change(function() {
+            displayLoadingBanner();
             this.form.submit();
         });
-    });
-    $(function() {
-        $('#show_this_year_of_elections_id').change(function() {
+        
+        {# end wv-103 #}
+        $('#show_statistics_id').change(function() {
+            displayLoadingBanner();
             this.form.submit();
         });
-    });
-    $(function() {
+
         $('#state_code_id').change(function() {
+            displayLoadingBanner();
             this.form.submit();
         });
+
+        $('form[name="organization_position_choose_election"]').submit(function(event) {
+            displayLoadingBanner();
+        });
+    });
+    $(function() {
```

Comment:
Can these function stubs be removed? I understand if you need to leave them in for a future change.